llvm/lib/ObjCopy/DXContainer/DXContainerReader.h

```diff
@@ -0,0 +1,34 @@
+//===- DXContainerReader.h --------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIB_OBJCOPY_DXCONTAINER_DXCONTAINERREADER_H
+#define LLVM_LIB_OBJCOPY_DXCONTAINER_DXCONTAINERREADER_H
+
+#include "DXContainerObject.h"
+
+namespace llvm {
+namespace objcopy {
+namespace dxc {
```

Comment:
nit: `dxc` is probably not the best namespace choice. Most of the binary format stuff we've put into the `dxbc` namespace which matches the 4-character magic number at the beginning of the file.